torch/distributed/_functional_collectives.py

```diff
@@ -402,6 +402,10 @@ def trigger_wait(self):
         wait_tensor(self.elem)
         return self
 
+    def wait(self) -> torch.Tensor:
```

Comment:
since these are 'public' apis (i mean, they are still in the _functional_collectives namespace so its not enforced yet) but they should still get legit docstrings.

Reply:
Oh yes, good call out! Let me add them in a follow up PR